eng/Versions.props

```diff
@@ -6,15 +6,14 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>2</PatchVersion>
-    <SdkBandVersion>6.0.100</SdkBandVersion>
-    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>
-    </PreReleaseVersionIteration>
+    <SdkBandVersion>6.0.200</SdkBandVersion>
```

Comment:
@steveisok this is an intentional change right?

Reply:
Yes I think we can take this into 6.0, I think this is just for testing purposes.

Reply:
No this actually gets embedded in the workloads manifest package id :)

Reply:
Yes, it's intentional.  Normally, we shouldn't have to bump it unless there are real changes, but the SDK has had problems rolling back workload installs if the manifest is older.  It doesn't hurt to bump in order to safeguard.